contents/handbook/strategy/strategy.md

```diff
@@ -5,31 +5,38 @@ showTitle: true
 ---
 
 ## TL;DR
+
 Our mission is to “Increase the number of successful products in the world”.
 
 We started by building an open-source product analytics platform with all the basic things you'd need to understand user behavior - funnels, trends, session recording, feature flags and many more features. Within a year we had thousands of customers using us and we started generating revenue.
 
-We focused on our paid product earlier this year. We quickly hit a milestone for the first 5 reference customers, and we've kept selling since. The next step on our journey is to figure out how to accelerate our top of funnel growth, whilst achieving larger order values from our Enterprise product.
+We focused on PostHog Scale (our first self-hosted paid product) earlier this year. We quickly hit a milestone for the first 5 reference customers, and we've kept selling since - exclusively focussing on inbound organic demand. We've had best in class net revenue retention from these customers and our PostHog Cloud users.
+
+The next step on our journey is to figure out how to accelerate our top of funnel growth, whilst achieving larger order values from our Enterprise product.
 
 ### Context
 
 We’ve grown a lot (10s of thousands of sign ups), but it’s clear that the many of our users use us *despite* a bad experience - complex deployment and maintenance, in particular. This is a great problem to have. It means we’re solving a hair on fire problem - product analytics that you can self host.
 
-For any company, nothing matters more than product market fit. If we get that right, it’ll be much easier for customer success, marketing or sales teams to succeed.
+Whilst PostHog's adoption is driven by one developer through our open source product, we're seeing those deployments converting steadily into paying relationships with PostHog Scale.
+
+We're aiming to keep expanding this adoption within the largest organizations using PostHog, whilst scaling our community growth.
 
 ### Mission
 
 **_“Increase the number of successful products in the world”_**
 
-### Long-term vision (for 2023)
+### Long-term vision (for 2026)
+
+**_“PostHog IPOs, becoming a standard tool for building a more successful product without losing control of user data”_**
```

Comment:
I'm strongly against this being our company vision, an IPO feels very far away from solving a real world problem for people (which is what motivates me - and many others to work here or elsewhere). I can imagine that we might IPO as a consequence of our success at effectively solving a real world problem, but I personally think it should not be the goal.

I'd be more comfortable with something like:

> "To be the standard tool for product analytics"

But I'm not certain there's any need to change the existing vision, it still seems pretty relevant to me?

Becoming publicly traded seems like a reasonable milestone to aim for as a company - since to achieve it we'll likely need to have significant community growth and revenue along the way. 

However there are many companies that have IPO'd with a lower market cap than our current valuation, so technically we might be able to IPO today - but it wouldn't be an indication that we've become a standard tool.

Reply:
> technically we might be able to IPO today

Bias for action?

Reply:
With the current statement:

"Everyone building a product has a clear path to making it successful without losing control of their data"

How specifically is this currently not achieved? From my view, I want to set clearer goals, and I felt our current Vision statement isn't helping enough here (other than saying we'll focus on self hosted, which is helpful).

Reply:
"Everyone building a product has a clear path to making it successful without losing control of their data"

Some examples of where we're a way off this today:
* **Everyone** - we have a long way to go with scalability for the largest businesses 
* **Clear path to success** - we have tools that give "clues" as to potential ways to improve your product (I.e. correlation analysis) but have a long way to go for people to know exactly what to do to make their product successful 

I'm all for making things clearer though - I'll have a think on Monday and suggest some potential ways to do that.

Reply:
I’ve spent a little while thinking about this and I feel the essence of the proposed vision is about becoming a standard tool. So I’d propose we simplify it to something like this:

> “To be the standard devtool for product analytics”

Rationale: 
*  **Specific:** There can be only one “standard” devtool for product analytics (this means not focussing on other analytics like building BI tools for accounting or logistics)
*  **Simple:** It contains a single concept which is easy to grasp and remember
* **Inspiring:** It would be really exciting to have most developers install PostHog on every product they build
*  **Verifiable:** We can validate that we’ve hit this vision through multiple sources (e.g. high traction on GitHub, high organic use in open source projects and tutorials, mentions on articles and review sites)
*  **Ambitious:** No-one has achieved this yet and there are a lot of product analytics tools - this is going to be tough
*  **Relevant:** In order to become the standard dev tool - we need to build something for developers (our current approach) which works for every organization (inc. Super-scale and those who need to control their data) - which are not well served today and we have traction with

We could make this more narrow and focus only on those who need to self host, but I feel it dilutes the ambition, since it could be argued that we are almost there already  e.g.

> “To be the standard self-hosted devtool for product analytics”

Also I’m totally up for the concept and the ambition of aiming for an IPO - but I think it should be a consequence of achieving the vision and not vision itself - perhaps we could add a milestones section like this below the vision?

e.g. 
>Annual Milestones:
>* 2026 - IPO at $xBn valuation
>* 2025 - $100M ARR, 100K Open source users
>* 2024 - $50M ARR…
>* 2023 - …
>* 2022 - …

Reply:
Unsolicited fly-by comment on:

> “To be the standard devtool for **product analytics**”

We do way more than the traditional definition of _product analytics_, so we should use something that doesn't put us in a box. In my book, that phrase still doesn't encompass session recording, feature flags, heatmaps, and other core feature pillars that we consider being our full stack product analytics.

If we angle ourselves only as _product analytics_, I think it creates too much association with the default competition in that space.

On our new [Product page](https://posthog.com/product), _product analytics_ is only one tab in our suite of features. The page headline alludes to something bigger:

> An ever-expanding suite of tools to build better products

Reply:
Fair feedback - in my mind I include all of those features as product analytics - but agree many wont.

> An ever-expanding suite of tools to build better products
I like this bit about building better products (over product analytics) - not keen on the ever-expanding bit as it feels by definition impossible to achieve and we might end up focussing on building new tools with diminishing returns over improving our existing ones to make them move valuable.

How about?
> “To be the standard devtool for building better products”

Reply:
I also dislike "product analytics" as the overall way we frame ourselves *internally*.

Customers show up asking for product analytics or session recording or feature flags or pipeline as that's how the market is educated. From a vision statement I think we can definitely go broader.

> "To be the standard devtool for building better products"

* It's simpler than what we have
* We need a ton of adoption to be successful, which this implies
* We are focussed on developers. A strong open source / self hosted offering is a core part of that.
* Isn't too limited based on just metrics. We might get here by going _far_ broader than we have today.


On the IPO - I think this is a very good goal to set as an outcome of doing the above. Becoming a standard tool would _massively_ help us get towards an IPO.